lib/msal-browser/docs/login-user.md

```diff
@@ -68,12 +68,19 @@ When a login call has succeeded, you can use the `getAllAccounts()` function to
 const myAccounts: AccountInfo[] = msalInstance.getAllAccounts();
 ```
 
-If you know the username of the account, you can also retrieve the account information by using the the `getAccountByUsername()` API:
+If you know the account information, you can also retrieve the account information by using the `getAccountByUsername()` or `getAccountById()` APIs:
```

Comment:
Can this be instead: 

If you know the account information, you can also retrieve the account information by using the `getAccountById()` or `getAccountByUsername()` APIs. We recommend `getAccountById()` as the default lookup API. Please note that `getAccountByUserName()` can be called for `idToken.preferred_name` for AAD and the first of `idToken.emails` claim for B2C if configured. 

... something that makes it obvious our preference and what they are choosing.

Reply:
Is the note below (Lines 80-83) insufficient or do you just want it moved up into this line?